### PR TITLE
Implemented power restore policy page

### PR DESCRIPTION
**Pull request by SteffiAntony**
Implemented power policy page
Jira: https://jsw.ibm.com/browse/PFEBMC-2404

### CODE REVIEW

File: src/views/Settings/PowerRestorePolicy/PowerRestorePolicy.vue [hunk 1]
```diff
@@ -1,131 +1,123 @@
 <template>
-  <b-container fluid="xl">
+  <div class="container" fluid="xl">
```
Comment (nabilpandiyalayil) [inline src/views/Settings/PowerRestorePolicy/PowerRestorePolicy.vue]: Use BContainer

File: src/views/Settings/PowerRestorePolicy/PowerRestorePolicy.vue [hunk 2]
```diff
@@ -1,131 +1,123 @@
...
-import Alert from '@/components/Global/Alert';
-import VuelidateMixin from '@/components/Mixins/VuelidateMixin.js';
-import BVToastMixin from '@/components/Mixins/BVToastMixin';
+<script setup>
+import { ref, onMounted, computed } from 'vue';
+import PageTitle from '../../../components/Global/PageTitle.vue';
+import { onBeforeRouteLeave } from 'vue-router';
+import Alert from '../../../components/Global/Alert.vue';
+import PowerPolicyStore from '../../../store/modules/Settings/PowerPolicyStore';
+import BootSettingsStore from '../../../store/modules/Operations/BootSettingsStore';
```
Comment (nabilpandiyalayil) [inline src/views/Settings/PowerRestorePolicy/PowerRestorePolicy.vue]: Import using '@'

**Review by nabilpandiyalayil — CHANGES_REQUESTED** [review]

File: src/views/Settings/PowerRestorePolicy/PowerRestorePolicy.vue [hunk 3]
```diff
@@ -1,131 +1,123 @@
...
-        </b-form-group>
-      </b-col>
-    </b-row>
+          ></BFormRadioGroup>
+        </BFormGroup>
+      </BCol>
+    </BRow>
 
-    <b-button
+    <BButton
```
Comment (Nikhil-Ashoka) [inline src/views/Settings/PowerRestorePolicy/PowerRestorePolicy.vue:41]: There is no gap between the options and the Save button. Please make this change

**Review by Nikhil-Ashoka — CHANGES_REQUESTED** [review]

File: src/views/Settings/PowerRestorePolicy/PowerRestorePolicy.vue [hunk 4]
```diff
@@ -1,131 +1,123 @@
...
-import Alert from '@/components/Global/Alert';
-import VuelidateMixin from '@/components/Mixins/VuelidateMixin.js';
-import BVToastMixin from '@/components/Mixins/BVToastMixin';
+<script setup>
+import { ref, onMounted, computed } from 'vue';
+import PageTitle from '@/components/Global/PageTitle.vue';
+import { onBeforeRouteLeave } from 'vue-router';
+import Alert from '@/components/Global/Alert.vue';
+import PowerPolicyStore from '@/store/modules/Settings/PowerPolicyStore';
+import BootSettingsStore from '@/store/modules/Operations/BootSettingsStore';
```
Comment (nabilpandiyalayil) [inline src/views/Settings/PowerRestorePolicy/PowerRestorePolicy.vue]: Import from '@/store'

**Review by nabilpandiyalayil — CHANGES_REQUESTED** [review]

**Review by Nikhil-Ashoka — APPROVED** [review]

**Review by Nikhil-Ashoka — APPROVED** [review]

**Review by nabilpandiyalayil — APPROVED** [review]

**Pull request merged**